### PR TITLE
IDEA: Add a local config file

### PR DESCRIPTION
**Pull request by bamadesigner**
## Description
I have BrowserSync enabled for a Rig project and the path to the certificate is local info so didn't want to commit it, which meant moving around the info constantly while I moved between branches and commits.

Enter this solution, which allows me to have a config.local.json file of untracked config info.

Thoughts?

Not necessary for 2.0. I can move around later if needed.

## List of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [ ] This pull request relates to a ticket.
- [X] This code is tested.
- [X] This change has been added to CHANGELOG.md
- [X] I want my code added to WP Rig.

### CODE REVIEW

**Comment by mor10** [comment]
How is this different from the current `config.json` file which allows you to add custom info?

**Comment by bamadesigner** [comment]
@mor10 This isn't tracked.

**Comment by bamadesigner** [comment]
@mor10 This isn't tracked, so you can include local info, like the path to your BrowserSync cert, or your own dev URLs, without having to force that on other devs or commit to a public repo.

**Comment by ataylorme** [comment]
I am +1 on config override that is not tracked. It can be used for proxy URL, certificate path, and other items specific to each person’s machine, rather than the project.

The existing config.json can still be used and tracked for project specific items, like namespace.

**Comment by mor10** [comment]
Pardon my currently-working-at-50%-capacity brain: With this change, we now have 3 config files?
- `config.default.json`
- `config.json` where the users make project-specific changes which are committed
- `config.local.json` where users make local-specific changes which are _not_ committed
Yes? If so, I'm good with the changes.

File: README.md [hunk 1]
```diff
@@ -102,7 +103,7 @@ WP Rig gives the developer an out of the box environment with support for modern
 
 Configuring the behavior of WP Rig is done by editing `./config/config.json`. Here the developer can set the theme name and theme author name (for translation files), and local server settings for BrowserSync. Additionally, compression of JavaScript and CSS files can be turned off for debugging purposes.
 
-Place your custom theme settings in `./config/config.json` to override default settings, located in `./config/config.default.json`.
+Place your custom theme settings in `./config/config.json` to override default settings, located in `./config/config.default.json`. Place local-only theme settings in `./config/config.local.json`, e.g. potentially sensitive info like the path to your BrowserSync certificate.
```
Comment (ataylorme) [inline README.md]: @bamadesigner can you expand this a bit? I would like to see the namespace options added as an example of what should go in `config.json`

**Comment by ataylorme** [comment]
Your summary is correct.

**Pull request merged**